test/controllers/tasks_controller_test.rb

```diff
@@ -79,24 +76,24 @@
 
       new_task = Task.find_by(name: task_hash[:task][:name])
       expect(new_task.description).must_equal task_hash[:task][:description]
-      expect(new_task.due_date.to_time.to_i).must_equal task_hash[:task][:due_date].to_i
-      expect(new_task.completed).must_equal task_hash[:task][:completed]
+      expect(new_task.completion_date).must_equal task_hash[:task][:completion_date]
```

Comment:
Since you don't have a `completion_date` field this should be `expect(new_task.current_status).must_equal task_hash[:task][:current_status]`

And you'll need to update the task_hash.